src/resources/views/fields/table.blade.php

```diff
@@ -104,7 +110,9 @@
             window.angularApp.controller('tableController', function($scope){
 
                 $scope.sortableOptions = {
-                    handle: '.sort-handle'
+                    handle: '.sort-handle',
+                    axis: 'y',
+                    helper: fixHelper,
```

Comment:
Can we just inline the function here rather than extracting it into its own function? Seems unnecessary in this instance

Reply:
Sure ! Just tested here hardcoding in my vendor and it works, not at my comp, can you please inline it? Something like this:

````javascript
handle: '.sort-handle',
axis: 'y',
helper: function(e, ui) {
	ui.children().each(function() {
		$(this).width($(this).width());
	});
	return ui;
},
````